lib/utils.pm

```diff
@@ -1825,4 +1827,28 @@ sub install_patterns {
     }
 }
 
+sub common_service_action {
+    my ($service, $type, $action) = @_;
+
+    if ($type eq 'SystemV') {
+        if ($action eq 'on') {
+            assert_script_run 'chkconfig ' . $service . ' on';
+            assert_script_run 'chkconfig ' . $service;
+        } else {
+            assert_script_run '/etc/init.d/' . $service . ' ' . $action;
+            assert_script_run 'service ' . $service . ' status';
+        }
+    }
+    elsif ($type eq 'Systemd') {
+        systemctl $action . ' ' . $service;
+        systemctl 'status ' . $service;
```

Comment:
It is better to verify the result after the action while not just show the status.